cli/tests/test_diff_command.rs

```diff
@@ -2274,6 +2275,59 @@ fn test_diff_external_tool() {
     ");
 }
 
+#[test]
+fn test_diff_do_chdir() {
+    let test_env = TestEnvironment::default();
+    test_env.run_jj_in(".", ["git", "init", "repo"]).success();
+    let work_dir = test_env.work_dir("repo");
+    work_dir.write_file("file1", "file1\n");
+
+    assert_eq!(
+        work_dir
+            .run_jj([
+                "diff",
+                "--tool=pwd",
+                "--config=merge-tools.pwd.diff-args=[]"
+            ])
+            .to_string()
+            .lines()
+            .next()
+            .unwrap(),
+        "$TEST_ENV/repo" // I don't know why pwd returns this, but it does
```

Comment:
(No action needed): `run_jj` substitutes some strings that change from one test run for another with a constant string. This applies to the directory for the test environment.

Reply:
Aha! Thanks, that explains that. But it doesn't appear to do that for Windows `pwd`, which is why that test fails.